scripts/build/bundler.js

```diff
@@ -176,13 +176,19 @@ function getWebpackConfig(bundle) {
       path: path.resolve(root, "dist"),
       filename: bundle.output,
       library: ["prettierPlugins", bundle.name],
-      libraryTarget: "umd"
+      libraryTarget: "umd",
+      // https://github.com/webpack/webpack/issues/6642
+      globalObject: "new Function('return this')()"
     },
     plugins: [
```

Comment:
In webpack 4 node env is [defined automatically](https://www.bennadel.com/blog/3598-webpack-4-automatically-makes-process-env-node-env-available-in-your-javascript.htm). Can remove the define plugin.

Reply:
removed